scorecards.go

```diff
@@ -28,16 +30,37 @@ type ScorecardConnection struct {
 	TotalCount int         `graphql:"totalCount"`
 }
 
-func (scorecard *Scorecard) AliasOwnerType() AliasOwnerTypeEnum {
-	return AliasOwnerTypeEnumScorecard
-}
-
 type ScorecardCategoryConnection struct {
 	Nodes      []Category `graphql:"nodes"`
 	PageInfo   PageInfo   `graphql:"pageInfo"`
 	TotalCount int        `graphql:"totalCount"`
 }
 
+func (scorecard *ScorecardId) ReconcileAliases(client *Client, aliasesWanted []string) error {
+	var allErrors, err error
+
+	aliasesToCreate, aliasesToDelete := ExtractAliases(scorecard.Aliases, aliasesWanted)
+	for _, alias := range aliasesToDelete {
+		err := client.DeleteAlias(AliasDeleteInput{
+			Alias:     alias,
+			OwnerType: AliasOwnerTypeEnumScorecard,
+		})
+		allErrors = errors.Join(allErrors, err)
+	}
```

Comment:
This seems to be identical code 3 or more times - good candidate for a helper/utility function that takes in `aliasesToDelete` and `OwnerType` and returns `error`